src/utils/ekubo_math.py

```diff
@@ -0,0 +1,161 @@
+import math
+
+def get_token_order(address1: str, address2: str) -> tuple:
+    """
+    Compares two token addresses and returns the addresses in the correct order, where token0_address < token1_address.
+
+    Parameters:
+    - address1 (str): The first token address (in hexadecimal format).
+    - address2 (str): The second token address (in hexadecimal format).
+
+    Returns:
+    - tuple: A tuple containing (token0_address, token1_address), where token0_address is the smaller of the two addresses.
+    """
+    # Convert the hexadecimal addresses to integers
+    address1_int = int(address1, 16)
+    address2_int = int(address2, 16)
+    
+    # Compare the integer values and return them in the correct order
+    if address1_int < address2_int:
+        return (address1, address2)  # address1 is token0_address, address2 is token1_address
+    else:
+        return (address2, address1)  # address2 is token0_address, address1 is token1_address
+
+def get_pool_fee_touse(pool_fee: float) -> int:
+    """
+    Calculate the pool fee scaled by 2^128 for blockchain usage.
+
+    Parameters:
+    - pool_fee (float): The pool fee as a percentage (e.g., for a 0.3% fee, pass 0.3).
+
+    Returns:
+    - int: The pool fee to use, scaled by 2^128.
+    """
+    return math.floor((pool_fee / 100) * 2**128)
+
+def get_tick_spacing_touse(tick_spacing: float) -> int:
+    """
+    Calculate the tick spacing to use based on logarithmic operations.
+
+    Parameters:
+    - tick_spacing (float): The tick spacing.
+
+    Returns:
+    - int: The tick spacing value, calculated and rounded appropriately.
+    """
+    return math.ceil(math.log(1 + tick_spacing / 100) / math.log(1.000001))
```

Comment:
Write test for this func using pytest. 

Use certain input values and check for correct expected values.